tests/test_sr/check.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+def freq(l):
+    """ return dictionnary of frequencies """
+    freq = {}
+    for item in l:
+        if (item in freq):
+            freq[item] += 1
+        else:
+            freq[item] = 1
+    return freq
+
+
+def read(name):
+    """ read filename and returns the distribution of outputs as a dictionnary """
+    with open(name, "r") as f:
+        data = [x.strip() for x in f.readlines()]
+        freqs = freq(data)
+    return freqs
+
+TOLERANCE=15
```

Comment:
How did you define Tolerance? I suppose it is related to the number of samples with a given confidence, maybe we can justify it in comments. (if I am right the number of 'samples' is the number of iteration of the loop in the C code (100)) 
We a single addition we can assume normal distribution and apply the confidence interval for it?

Reply:
I added a comment. Thanks.